langchain/src/tools/google_calendar/prompts/create-event-prompt.ts

```diff
@@ -0,0 +1,56 @@
+export const CREATE_EVENT_PROMPT = `
+Date format: YYYY-MM-DDThh:mm:ss+00:00
+Based on this event description: "Joey birthday tomorrow at 7 pm",
+output a json of the following parameters: 
+Today's datetime on UTC time 2023-05-02T10:00:00+00:00, it's Tuesday and timezone
+of the user is -5, take into account the timezone of the user and today's date.
+1. event_summary 
+2. event_start_time 
+3. event_end_time 
+4. event_location 
+5. event_description 
+6. user_timezone
+event_summary:
+{{
```

Comment:
See above

Reply:
This is passed here: https://github.com/nearform/langchainjs/blob/main/langchain/src/tools/google_calendar/commands/run-create-events.ts#L71

Reply:
In this case using an output parser would make sense I think?